app/models/drupal_node.rb

```diff
@@ -473,11 +473,9 @@ def tagnames
     self.tags.collect(&:name)
   end
 
-  # increment view count
+  # view counter is handled by controller
```

Comment:
@yukiisbored I think this method can be safely removed now since view counts will be done by impressionist now. But after removing run the tests to be extra sure which will be done by the CI anyways.